internal/pkg/cli/task_run.go

```diff
@@ -79,6 +79,11 @@ Select %s to run the task in your default VPC instead of any existing applicatio
 Select %s to run the task in your default VPC instead of any existing environment.`, color.Emphasize(appEnvOptionNone))
 )
 
+var (
+	taskSecretsPermissionPrompt     = "\nDo you grant permission to the ECS/Fargate agent for these secrets? "
```

Comment:
Do we need the "\n"?

Reply:
After listing out the secrets to which we need permissions it's good to have one "\n" else everything looks too close to each other.

Reply:
nit:
```suggestion
	taskSecretsPermissionPrompt     = "\nDo you grant permission to the ECS/Fargate agent for these secrets?"
``` 
I think `prompt.Confirm` seems to already creates a new line before the prompt. We probably don't need the extra new line. Adding together it's three new lines in a row, one at the end of the last secret (`"* secret"`), one imposed by `prompt.Confirm`, one prepended here.